js/game_state.js

```diff
@@ -298,9 +298,9 @@ var cGameState = function (level) {
  * @param {string[][]} to  - Board to copy cells to.
  */
 function cloneCells(from, to) {
```

Comment:
in `cloneCells` the first argument is `from `and 2nd is `to `, but while invoking we are passing it the other way which is leading to failure